phpstan.neon

```
@@ -60,6 +60,9 @@ parameters:
     excludePaths:
         - src/cli/src/Flow/CLI/Command/*
         - src/core/etl/src/Flow/ETL/Formatter/ASCII/ASCIITable.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelectorAll.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelector.php
+        - src/core/etl/src/Flow/ETL/Row/Entry/HTMLEntry.php
```

how many suppression those would require in file?
(I'm not home today, so sorry if I'm asking redundant questions, not everything is fully visible from phone 😅)

---

Few are some not really fixable by comment as they are generics related that is saying given class doesn't exists 🫣

---

Hmm doesn't phpstan allow to suppress specific issues in config for specific file/regex?

---

It's called baseline which you don't like 🤣

---

> It's called baseline which you don't like 🤣

It's called "knowing the tools you are using", more details in [documentation](https://phpstan.org/user-guide/ignoring-errors#ignoring-in-configuration-file) 

Errors related to missing \DOM\HTMLDocument can be suppressed like in the example below. 

```
parameters:
	ignoreErrors:
		-
			messages:
				- '#Call to an undefined method [a-zA-Z0-9\\_]+::doFooFoo\(\)#'
				- '#Call to an undefined method [a-zA-Z0-9\\_]+::doFooBar\(\)#'
			path: other/dir/AnotherFile.php
```

---

I know that tool and what baseline is (which is a list of such ignored errors, nothing more). Done.